package.nls.ja.json

```diff
@@ -0,0 +1,71 @@
+  "configuration.SAS.connectionProfiles.profiles": "SAS 接続プロファイル",
+  "configuration.SAS.connectionProfiles.profiles.autoExec": "SAS 接続 AutoExec",
+  "configuration.SAS.connectionProfiles.profiles.autoExec.filePath": "SAS 接続 AutoExec ファイルパス",
+  "configuration.SAS.connectionProfiles.profiles.autoExec.line": "SAS 接続 AutoExec 行",
+  "configuration.SAS.connectionProfiles.profiles.autoExec.type": "SAS 接続 AutoExec 種類",
+  "configuration.SAS.connectionProfiles.profiles.clientId": "SAS Viya クライアント ID",
+  "configuration.SAS.connectionProfiles.profiles.clientSecret": "SAS Viya クライアントシークレット",
+  "configuration.SAS.connectionProfiles.profiles.com.host": "SAS COM 接続ホスト",
+  "configuration.SAS.connectionProfiles.profiles.connectionType": "SAS プロファイル接続タイプ",
+  "configuration.SAS.connectionProfiles.profiles.context": "SAS Viya コンテキスト",
+  "configuration.SAS.connectionProfiles.profiles.endpoint": "SAS Viya 接続プロファイルエンドポイント",
+  "configuration.SAS.connectionProfiles.profiles.name": "SAS 接続プロファイル名",
+  "configuration.SAS.connectionProfiles.profiles.sasOptions": "SAS 接続 SAS オプション",
+  "configuration.SAS.connectionProfiles.profiles.ssh.host": "SAS SSH 接続 SSH ホスト",
+  "configuration.SAS.connectionProfiles.profiles.ssh.port": "SAS SSH 接続ポート",
+  "configuration.SAS.connectionProfiles.profiles.ssh.saspath": "SAS SSH 接続の実行可能パス",
+  "configuration.SAS.connectionProfiles.profiles.ssh.username": "SAS SSH 接続ユーザー名",
+  "configuration.SAS.connectionProfiles.profiles.tokenFile": "SAS Viya トークンファイル",
+  "configuration.SAS.connectionProfiles.profiles.username": "SAS Viya ユーザー ID",
+  "configuration.SAS.flowConversionMode": "ノートブックの変換モードを選択する",
+  "configuration.SAS.flowConversionModeNode": "各ノートブックのセルをノードに変換する",
+  "configuration.SAS.flowConversionModeSwimlane": "各ノートブックのセルをスイムレーンに変換する",
+  "configuration.SAS.results.html.enabled": "ODS HTML5 出力の有効化/無効化を設定する",
+  "configuration.SAS.results.html.style": "ODS HTML5 結果のスタイルを指定する",
+  "configuration.SAS.results.html.style.(auto)": "カラーテーマに最も近いスタイルを拡張機能に選択させます。",
+  "configuration.SAS.results.html.style.(server default)": "デフォルトは、SAS Server で構成されたスタイルです。",
+  "configuration.SAS.results.sideBySide": "結果をコードの横に表示する",
+  "configuration.SAS.results.singlePanel": "単一パネルを再利用して結果を表示する",
+  "configuration.SAS.userProvidedCertificates": "信頼された証明機関 (CA) 証明書ファイルを指定する",
+  "notebooks.SAS.htmlRenderer": "SAS HTML レンダラー",
+  "notebooks.SAS.logRenderer": "SAS ログレンダラー",
+  "notebooks.SAS.sasNotebook": "SAS ノートブック",
+  "themes.SAS.dark": "SAS Dark",
+  "themes.SAS.highContrast": "SAS High Contrast",
+  "themes.SAS.light": "SAS Light",
+  "views.SAS.explorer": "エクスプローラー",
+  "views.SAS.libraries": "ライブラリ",
+  "views.SAS.signIn": "サインイン",
+  "views.SAS.unsupportedConnection": "この接続では、SAS Extension for Visual Studio Code 内の SAS コンテンツナビゲーションがサポートされていません。ファイルにアクセスできるのは、エクスプローラービューのみです。",
+  "views.SAS.welcome": "SAS Extension for Visual Studio Code を使用するには、SAS にサインインする必要があります。\n [Sign In](コマンド: SAS.authorize)"

```

Comment:
Hi, please do NOT translate `command:` as it's part of the hyperlink syntax to trigger the command. Thanks.